script/Arcueid/Day 4/04_00_ARC04_6_2C.txt

```diff
@@ -51,7 +51,7 @@ At some point, her phrasing changed from 'my enemy' to 'our enemy'. I wanted to
 [sha:cabd02be85d305124e7bfb8ccae09c80f840e6c2]{
 -- Page 532, Offset 7011.
 -- 魔術の果てに不老になったものか、血を吸われて下僕になったものとがいる。
-Some are immortalized by magecraft, while others have their blood sucked and become servants.
+Some are immortalized by %{i}magecraft%{/i}, while others have their blood sucked and become servants.
```

Comment:
no need to italicize this

Reply:
i have been italicizing terms when they're first mentioned like this one. is it bad?

Reply:
No that's totally reasonable, but this isn't really what I'd consider a lore term. It's more just like... a word I guess? I don't really consider it italics-worthy at the very least.

Reply:
alright i thought it's a nasuverse thing but it's actually a common term used by other franchise